gulpfile.js

```diff
@@ -555,15 +555,17 @@ const RELEASE_SOURCES = [
 const RELEASE_LIB = [
     'dist/node.*',
     'dist/worker-*',
+    'dist/web.*'
 ];
```

Comment:
What about `web-offline.js`?

Reply:
Hmm, good point, thanks @sisou .

I can see two scenarios (but there are probably more) where someone would want to have offline support:

1. On an air gapped system which is never connected to the Internet.
1. On a regular system where the user can't or doesn't want to be online sometimes.

For the first option we should actually have a completely separate package, for the second option I guess it makes sense to include it on this package.

What's the intended usage of this file (i.e. how is an user of the packages expected to use this file)?

Reply:
The file is also included in the CDN and is for example used in the keyguard, where we don't need networking capabilities.